src/common/transformations/tests/op_conversions/convert_gather_to_compressed_test.cpp

```diff
@@ -225,3 +225,50 @@ TEST_F(TransformationTestsF, ConvertGatherToCompressedMultiOutput) {
         model_ref = std::make_shared<ov::Model>(ov::NodeVector{gather_compressed}, ov::ParameterVector{input1, input2});
     }
 }
+
+// In compressed FP16/BF16 weight case, it transform gather node with constant weight decompression pattern(FP16/BF16 +
+// convert(FP32)) to gather node with compressed(FP16/BF16) weight, and move decompression after gather node, so
+// GatherCompressed node should not generated.
```

Comment:
Small comment remarks (optional):
```suggestion
// In compressed FP16/BF16 weight case, gather node with constant weight decompression pattern (FP16/BF16 +
// convert(FP32)) is transformed to gather node with compressed (FP16/BF16) weights, and decompression convert is moved after gather node, so GatherCompressed node should not be generated.
```

Reply:
applied, thanks Vladislav!